pkg/maps/lbmap/types.go

```diff
@@ -11,6 +11,24 @@ import (
 	"github.com/cilium/cilium/pkg/loadbalancer"
 )
 
+// LBMap is the interface describing methods for manipulating service maps.
+type LBMap interface {
+	UpsertService(*UpsertServiceParams) error
+	UpsertMaglevLookupTable(uint16, map[string]loadbalancer.BackendID, bool) error
+	IsMaglevLookupTableRecreated(bool) bool
+	DeleteService(loadbalancer.L3n4AddrID, int, bool, loadbalancer.SVCNatPolicy) error
+	AddBackend(loadbalancer.Backend, bool) error
+	UpdateBackendWithState(loadbalancer.Backend) error
+	DeleteBackendByID(loadbalancer.BackendID) error
+	AddAffinityMatch(uint16, loadbalancer.BackendID) error
+	DeleteAffinityMatch(uint16, loadbalancer.BackendID) error
+	UpdateSourceRanges(uint16, []*cidr.CIDR, []*cidr.CIDR, bool) error
+	DumpServiceMaps() ([]*loadbalancer.SVC, []error)
+	DumpBackendMaps() ([]*loadbalancer.Backend, error)
+	DumpAffinityMatches() (BackendIDByServiceIDSet, error)
+	DumpSourceRanges(bool) (SourceRangeSetByServiceID, error)
+}
```

Comment:
Can you explain this move? To me it's odd to see an interface being part of the package where its implementation lives. Shouldn't an interface be defined somewhere else? See [1] to continue.